test/persistence-hooks.suite.js

```diff
@@ -1287,6 +1515,42 @@ module.exports = function(dataSource, should) {
           });
       });
 
+      it('triggers `persist` hook on update', function(done) {
+        TestModel.observe('persist', pushContextAndNext());
+
+        TestModel.updateAll(
+          { where: { name: existingInstance.name } },
+          { name: 'changed' },
+          function(err, instance) {
+            if (err) return done(err);
+
+            observedContexts.should.eql(aTestModelCtx({
+              data: { name: 'changed' },
+              currentInstance: { name: 'changed' },
```

Comment:
As I mentioned earlier, there should be no `currentInstance` in the context provided for `updateAll`.